test/blackbox/common/DDSBlackboxTestsStatistics.cpp

```diff
@@ -193,3 +193,56 @@ TEST(DDSStatistics, simple_statistics_datareaders)
 
 #endif // FASTDDS_STATISTICS
 }
+
+TEST(DDSStatistics, simple_statistics_second_writer)
+{
+#ifdef FASTDDS_STATISTICS
+
+    auto transport = std::make_shared<UDPv4TransportDescriptor>();
+    auto domain_id = GET_PID() % 230;
```

Comment:
```suggestion
    auto domain_id = GET_PID() % 100;
```

Using higher `domain_id`s causes in Windows that the ports that are tried to use are already in use by the operating system.